.github/workflows/bundlers.yml

```diff
@@ -115,11 +116,78 @@ jobs:
       - name: Extract tarball
         run: tar -xzf /tmp/packages/uppy-${{ github.sha }}.tgz --strip-components 1
       - name: Add Webpack as a dev dependency
-        run: npm i --save-dev webpack-cli webpack@${{matrix.webpack-version}}
+        run: npm i --save-dev webpack-cli webpack@${{matrix.bundler-version}}
       - name: Create Webpack config file
         run: echo 'module.exports={mode:"production",target:"web",entry:"./index.mjs"}' > webpack.config.js
       - name: Bundle
         run: npx webpack
-  # ESBuild: We are already testing it when generating the actual bundle.
+
+  parcel:
+    needs: isolate_uppy
+    name: Parcel
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bundler-version: [latest]
+    steps:
+      - name: Download uppy tarball
+        uses: actions/download-artifact@v3
+        with:
+          path: /tmp/
+      - name: Extract tarball
+        run: tar -xzf /tmp/packages/uppy-${{ github.sha }}.tgz --strip-components 1
+      - name: Fix package.json to work with Parcel
+        run: |
+          node <<'EOF'
+            const pkg = require('./package.json');
+            delete pkg.main
+            delete pkg.types
+            pkg.module = 'output.mjs'
+            require('node:fs').writeFileSync('./package.json', JSON.stringify(pkg));
+          EOF
+      - name: Add Parcel as a dev dependency
+        run: npm i --save-dev parcel@${{matrix.bundler-version}}
+      - name: Bundle
+        run: npx parcel build index.mjs
+
+  vite:
+    needs: isolate_uppy
+    name: Vite
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bundler-version: [latest]
+    steps:
+      - name: Download uppy tarball
+        uses: actions/download-artifact@v3
+        with:
+          path: /tmp/
+      - name: Extract tarball
+        run: tar -xzf /tmp/packages/uppy-${{ github.sha }}.tgz --strip-components 1
+      - name: Add Vite as a dev dependency
+        run: npm i --save-dev vite@${{matrix.bundler-version}}
+      - name: Create index.html
+        run: echo '<!doctype html><html><head><script type="module" src="./index.mjs"></script></head></html>' > index.html
+      - name: Bundle
+        run: npx vite build
+
+  esbuild:
```

Comment:
Vite uses esbuild under the hood and we use Vite always in development so perhaps those aren't strictly required. But it doesn't hurt either, as long as it doesn't stall CI too long.

Reply:
Yeah I thought that originally, but actually we have custom config for our dev setup, the idea is to test the defaults – or as close as the defaults as possible so users can see what's the minimal config change they need to do to make Uppy works for their setup.